Sources/MessagingPush/Service/Request/MetricRequest.swift

```diff
@@ -8,15 +8,21 @@ public enum Metric: String, Codable {
 
 // https://customer.io/docs/api/#operation/pushMetrics
 internal struct MetricRequest: Codable {
-    let deliveryID: String
+    let deliveryId: String
     let event: Metric
     let deviceToken: String
     let timestamp: Date
 
     enum CodingKeys: String, CodingKey {
-        case deliveryID = "delivery_id"
+        case deliveryId
         case event
-        case deviceToken = "device_id"
+        case deviceToken = "deviceId"
```

Comment:
Are you sure key is `deviceId` because previously it was `device_id` and also in the test case i see `device_id` as expected json.
`{"delivery_id":"123","device_id":"234","event":"opened","timestamp":1642018466}`

Reply:
Yes I am sure. The test functions test the functionality works as intended. 

I tried to explain why `deviceId` works [in the PR description](https://github.com/customerio/customerio-ios/pull/111#issue-1100764293)

Reply:
:grimacing: not a fan of this sort of magic, but seems from some research to be the swift-y way to do it, given that the `convertFromSnakeCase` is a builtin